fastapi_sso/sso/naver.py

```diff
@@ -23,4 +23,9 @@ async def get_discovery_document(self) -> DiscoveryDocument:
         }
 
     async def openid_from_response(self, response: dict, session: Optional["httpx.AsyncClient"] = None) -> OpenID:
-        return OpenID(display_name=response["properties"]["nickname"], provider=self.provider)
+        return OpenID(
+            id=response["response"]["id"],
+            display_name=response["response"]["nickname"],
+            picture=response["response"]["profile_image"] or "",
```

Comment:
I'd prefer `None` instead of `""` if the profile image isn't available, could you please change this?

Reply:
Also I just noticed the tests are failing, would you be able to fix the test response for Naver?